.github/workflows/test.yml

```diff
@@ -33,30 +33,39 @@ jobs:
   tests:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        os: [macos-13, ubuntu-22.04]
+        os: [macos-13, ubuntu-22.04, windows-2022]
         rust: [stable, 1.71.0]
         test: ['std', 'no-std', 'examples']
         include:
           - cache: stable
             rust: stable
           - cache: 1-71-0
             rust: 1.71.0
-          - coverage-flags: COVERAGE=1
+          - os: ubuntu-22.04
+            coverage-flags: COVERAGE=1
             rust: stable
             test: std
-            os: ubuntu-22.04
-          - wgpu-flags: DISABLE_WGPU=1
+          - os: macos-13
             rust: stable
             test: std
-            os: macos-13
+          - os: windows-2022
+            # powershell format
+            wgpu-flags: '$env:DISABLE_WGPU = "1";'
```

Comment:
Since we use `shell: bash` for some Windows steps, do you think we should use the `bash` syntax here or better to leave the `Powershell` one?

Reply:
I switched to bash shell.